docs/source/daml2ts/index.rst

```diff
@@ -203,10 +203,11 @@ the generated TypeScript will consist of a type declaration and the definition o
 
    type Color = | 'Red' | 'Blue' | 'Yellow'
```

Comment:
```suggestion
   type Color = 'Red' | 'Blue' | 'Yellow'
```
That's easier on the eye if you put it all in one line, which makes a lot of sense here.